README.md

```diff
@@ -4,54 +4,51 @@
-- **Coding style**: Automatic code style formatting using [Black](https://github.com/psf/black), [Flake8](https://github.com/pycqa/flake8), and [isort](https://github.com/PyCQA/isort).
-- **Static type check**: Static type checking with [Mypy](https://github.com/python/mypy) to assist in finding bugs.
-- **pytest**: Easily add test code using [pytest](https://github.com/pytest-dev/pytest).
-- **GitHub features**: Some useful features, [workflow](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions) for style check and test for pull request, [issue template](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository), etc. are pre-implemented.
+- **Containerization**: Dependency minimization and code portability enhancement using [Docker](https://www.docker.com/).
+- **Virtual Environment / Package Management**: Development environment reproducibility ensured by [Poetry](https://python-poetry.org/).
+- **Coding Style**: Automatic code linting and formatting with [Ruff](https://docs.astral.sh/ruff/).
+- **Static Type Checking**: Early bug detection assisted by [Mypy](https://github.com/python/mypy).
+- **Testing**: Testing simplification achieved through [pytest](https://github.com/pytest-dev/pytest).
+- **GitHub Integration**: Integration features including [GitHub Actions workflows](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions), issue templates, and more.
 
-Please check [the slide format resources about Ascender (Japanese)](https://cvpaperchallenge.github.io/Britannica/ascender/ja) too.
+Please also view [resources about Ascender (in Japanese)](https://cvpaperchallenge.github.io/Britannica/ascender/ja).
 
 ## Project Organization
 
 ```
-    ├── .github/           <- Settings for GitHub.
+    ├── .github/               <- GitHub settings.
     │
-    ├── data/              <- Datasets.
+    ├── data/                  <- Datasets.
     │
-    ├── environments/       <- Provision depends on environments.
+    ├── environments/          <- Environment-specific configurations.
     │
-    ├── models/            <- Pretrained and serialized models.
+    ├── models/                <- Pretrained and serialized models.
     │
-    ├── notebooks/         <- Jupyter notebooks.
+    ├── notebooks/             <- Jupyter notebooks.
     │
-    ├── outputs/           <- Outputs.
+    ├── outputs/               <- Outputs.
     │
-    ├── src/               <- Source code. This sould be Python module.
+    ├── src/                   <- Python Source code.
     │
-    ├── tests/             <- Test codes.
+    ├── tests/                 <- Test code.
     │
-    ├── .flake8            <- Setting file for Flake8.
+    ├── .flake8                <- Flake8 configuration.
```

Comment:
We will remove `.flake8` file in this PR, right?

Reply:
Thank you for your suggestion.
This is so true.